src/main/java/cl/transbank/common/Option.java

```diff
@@ -0,0 +1,18 @@
+package cl.transbank.common;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+public abstract class Option {
+    public abstract String getCommerceCode();
```

Comment:
`commerce_code` and `api_key` could be implemented directly in this abstract class.